### PR TITLE
Tcp and Serial connection Support, Fix Install Errors on Windows

### PR DESCRIPTION
**Pull request by Rjasuja**
Tcp Connection Support : Default Ip as 127.0.0.1 and port as 5760
Serial Connection Support(only for Linux) : Default Port as ttyS0 and Baudrate as 9600.

Fix Install Errors For Dronecore Library and Example
Fix install path for Dronecore library on Windows in CMakeLists
Fix Error flags for Takeoff_and_land Example on Windows

### CODE REVIEW

File: include/dronecore.h [hunk 1]
```diff
@@ -77,6 +78,40 @@ class DroneCore
     ConnectionResult add_udp_connection(int local_port_number);
 
     /**
+     * @brief Adds a TCP connection with the default arguments.
+     *
+     * This will connect to local TCP port 5760.
+     *
+     * @return The result of adding the connection.
+     */
+    ConnectionResult add_tcp_connection();
```
Comment (hamishwillee) [inline include/dronecore.h]: Just For my information, why don't we have single function with default arguments for these connection APIs?

**Comment by hamishwillee** [comment]
Docs text looks good to me (made some minor updates). Just for my information - why don't we have single function with default arguments for these connection APIs (rather than two methods).

**Comment by hamishwillee** [comment]
When is this planned to be merged?

**Comment by julianoes** [comment]
Rebased and fixed in #59.

**Pull request closed without merging**